cdc/sorter/memory/entry_sorter.go

```diff
@@ -159,12 +159,27 @@ func (es *EntrySorter) Output() <-chan *model.PolymorphicEvent {
 }
 
 func eventLess(i *model.PolymorphicEvent, j *model.PolymorphicEvent) bool {
+	return Compare(i, j)
+}
+
+// Compare compares two events by CRTs, Resolved, StartTs, Put/Delete order.
+func Compare(i *model.PolymorphicEvent, j *model.PolymorphicEvent) bool {
```

Comment:
Maybe this function belongs to `model` together with `PolymorphicEvent`?

Reply:
Done, PTAL!